etc/kdenlive.profile

```diff
@@ -0,0 +1,27 @@
+# Firejail profile for kdenlive
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/kdenlive.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+
+caps.drop all
+net none
+nodvd
+nogroups
+nonewprivs
+noroot
+notv
+protocol unix,inet,inet6
```

Comment:
:+1: for `protocol unix,inet,inet6` and `net none` combination to keep libdc1394 initializing

Reply:
Oh lol, I didn't do that on purpose. So firewire needs inet?

Reply:
With `protocol unix` I get `libdc1394 error: Failed to initialize libdc1394`. 
However, I don't have a device suitable for testing, so to be honest I don't know how big the real world impact is. 
In any case `net none` should mitigate most of the risks that come with a more permissive protocol filter.